src/components/app/App.tsx

```diff
@@ -1,30 +1,55 @@
 import Main from '../main/Main';
 
 import styles from './App.css';
+import commonStyles from '../../styles/commonStyles.css';
 
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { Routes, Route, BrowserRouter } from 'react-router-dom';
 
 import { Layout } from 'components/Layout';
 import { Countries } from 'components/countries/Countries';
 import { Details } from 'components/details/Details';
 import { Feedback } from 'components/feedback/Feedback';
 import { Page404 } from 'components/page404/Page404';
+import { setInLocalStorage, getFromLocalStorage } from 'services/localStorage';
 
 const App: FC = () => {
+  const [token, setToken] = useState<string>('');
+
+  async function getToken(): Promise<string> {
+    const result = await fetch('http://127.0.0.1:8081/token');
```

Comment:
does it work now?

Reply:
fixed it and it worked
localhost = 127.0.0.1 or no?

Reply:
Yes it is, but it might be a bit confusing :)